js/coinex.js

```diff
@@ -3527,6 +3527,72 @@ module.exports = class coinex extends Exchange {
         };
     }
 
+    async fetchFundingRates (symbols = undefined, params = {}) {
+        /**
+         *  @method
+         * @name coinex#fetchFundingRates
+         * @description fetch the current funding rates
+         * @param {array} symbols unified market symbols
+         * @param {object} params extra parameters specific to the coinex api endpoint
+         * @returns {array} an array of [funding rate structures]{@link https://docs.ccxt.com/en/latest/manual.html#funding-rate-structure}
+         */
+        await this.loadMarkets ();
```

Comment:
@cheunhong by default coinex loads only spot markets, so the logic above won't work properly because we will mix spot and swap markets. 

I will try to refactor it a bit